src/hotspot/share/adlc/main.cpp

```diff
@@ -491,8 +491,6 @@ int get_legal_text(FileBuff &fbuf, char **legal_text)
   return (int) (legal_end - legal_start);
 }
 
-#if !defined(_WIN32) || defined(_WIN64)
```

Comment:
Removing the conditionalization is fine for this change.  But see also 
https://bugs.openjdk.org/browse/JDK-8342639
I've added a note there that this change removed the conditionalization.

Reply:
I'm glad you're giving some TLC to adlc. It is in desperate need of it.

Reply:
TLC?

Reply:
https://www.vocabulary.com/dictionary/TLC